tests/end2end/features/spawn.feature

```diff
@@ -10,7 +10,11 @@ Feature: :spawn
 
     Scenario: Starting a userscript which doesn't exist
         When I run :spawn -u this_does_not_exist
-        Then the error "Error while spawning userscript: The process failed to start." should be shown
+        Then regex "Userscript 'this_does_not_exist' not found in userscript directory '(.*)'( or '(.*)')*" should be logged
```

Comment:
Ah, I didn't know `*` worked in those. I tried `(.*)` with the `message` `should be shown` clause